Command/BaseBootstrapSymlinkCommand.php

```diff
@@ -62,7 +62,11 @@ public static function checkSymlink($symlinkTarget, $symlinkName, $forceSymlink
                 if (!$forceSymlink) {
                     throw new \Exception(sprintf('Symlink "%s" points to "%s" instead of "%s"', $symlinkName, $linkTarget, $symlinkTarget));
                 }
-                unlink($symlinkName);
+                if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN' && is_dir($linkTarget)) {
```

Comment:
Right. I mistyped the ID :)

And I think the purpose for that check is a bit different, while I took the check from the remove function of the filesystem component.